From 835891a5e140e9095e6496e1c4752499feff0d27 Mon Sep 17 00:00:00 2001
From: PatrikLundell <j.patrik.r.lundell@gmail.com>
Date: Sun, 25 Aug 2024 02:56:08 +0200
Subject: [PATCH 117/125] Made human products get human 'vitamin' and vitamin
 inheritance (#75918)

* Made human products get human 'vitamin' and vitamin inheritance

* Update data/json/items/comestibles/carnivore.json

Co-authored-by: Karol1223 <68503002+Karol1223@users.noreply.github.com>

---------

Co-authored-by: Karol1223 <68503002+Karol1223@users.noreply.github.com>
---
 data/json/items/comestibles/carnivore.json | 1192 ++++++++++++++------
 data/json/items/resources/bone.json        |    2 +-
 2 files changed, 847 insertions(+), 347 deletions(-)

diff --git a/data/json/items/comestibles/carnivore.json b/data/json/items/comestibles/carnivore.json
index ff83138608..b3496135ab 100644
--- a/data/json/items/comestibles/carnivore.json
+++ b/data/json/items/comestibles/carnivore.json
@@ -4,12 +4,12 @@
     "type": "COMESTIBLE",
     "comestible_type": "FOOD",
     "name": { "str_sp": "meat/fish" },
-    "material": [ "flesh" ],
+    "material": ["flesh"],
     "symbol": "%",
     "color": "red",
     "parasites": 32,
-    "vitamins": [ [ "meat_allergen", 1 ] ],
-    "flags": [ "RAW" ]
+    "vitamins": [["meat_allergen", 1]],
+    "flags": ["RAW"]
   },
   {
     "id": "froglegs",
@@ -23,32 +23,41 @@
     "price": "10 USD",
     "price_postapoc": "15 cent",
     "spoils_in": "12 hours",
-    "flags": [ "SMOKABLE", "RAW" ],
+    "flags": ["SMOKABLE", "RAW"],
     "smoking_result": "froglegs_smoked",
-    "vitamins": [ [ "calcium", 18 ], [ "iron", 1.5 ] ],
+    "vitamins": [
+      ["calcium", 18],
+      ["iron", 1.5]
+    ],
     "calories": 73
   },
   {
     "id": "froglegs_cooked",
     "copy-from": "froglegs",
     "type": "COMESTIBLE",
-    "name": { "str": "pair of cooked frog legs", "str_pl": "pairs of cooked frog legs" },
+    "name": {
+      "str": "pair of cooked frog legs",
+      "str_pl": "pairs of cooked frog legs"
+    },
     "parasites": 0,
     "spoils_in": "1 day",
-    "flags": [ "EATEN_HOT" ],
+    "flags": ["EATEN_HOT"],
     "price_postapoc": "1 USD"
   },
   {
     "id": "froglegs_smoked",
     "copy-from": "froglegs_cooked",
     "type": "COMESTIBLE",
-    "name": { "str": "pair of smoked frog legs", "str_pl": "pairs of smoked frog legs" },
+    "name": {
+      "str": "pair of smoked frog legs",
+      "str_pl": "pairs of smoked frog legs"
+    },
     "color": "brown",
     "spoils_in": "42 days",
     "price_postapoc": "2 USD",
     "quench": -1,
     "fun": 5,
-    "flags": [ "EATEN_HOT", "SMOKED", "SMOKABLE" ],
+    "flags": ["EATEN_HOT", "SMOKED", "SMOKABLE"],
     "smoking_result": "dry_froglegs"
   },
   {
@@ -63,9 +72,13 @@
     "price": "5 USD",
     "price_postapoc": "50 cent",
     "spoils_in": "12 hours",
-    "flags": [ "SMOKABLE", "RAW" ],
+    "flags": ["SMOKABLE", "RAW"],
     "smoking_result": "fish_smoked",
-    "vitamins": [ [ "vitC", 2 ], [ "calcium", 6 ], [ "iron", 14 ] ],
+    "vitamins": [
+      ["vitC", 2],
+      ["calcium", 6],
+      ["iron", 14]
+    ],
     "calories": 260
   },
   {
@@ -75,18 +88,29 @@
     "name": { "str": "scrap of fish", "str_pl": "scraps of fish" },
     "description": "A tiny scrap of edible fish.  It's not much, but it'll do in a pinch.",
     "cooks_like": "fish_scrap_cooked",
-    "proportional": { "weight": 0.1, "volume": 0.1, "price": 0.1, "calories": 0.1 },
-    "delete": { "flags": [ "SMOKABLE" ] },
-    "vitamins": [ [ "calcium", 1 ], [ "iron", 1 ] ]
+    "proportional": {
+      "weight": 0.1,
+      "volume": 0.1,
+      "price": 0.1,
+      "calories": 0.1
+    },
+    "delete": { "flags": ["SMOKABLE"] },
+    "vitamins": [
+      ["calcium", 1],
+      ["iron", 1]
+    ]
   },
   {
     "id": "fish_scrap_cooked",
     "copy-from": "fish_scrap",
     "type": "COMESTIBLE",
-    "name": { "str": "cooked scrap of fish", "str_pl": "cooked scraps of fish" },
+    "name": {
+      "str": "cooked scrap of fish",
+      "str_pl": "cooked scraps of fish"
+    },
     "parasites": 0,
     "spoils_in": "1 day",
-    "flags": [ "NUTRIENT_OVERRIDE" ],
+    "flags": ["NUTRIENT_OVERRIDE"],
     "calories": 51
   },
   {
@@ -99,13 +123,16 @@
     "spoils_in": "1 day",
     "price_postapoc": "1 USD",
     "parasites": 0,
-    "flags": [ "EATEN_HOT" ]
+    "flags": ["EATEN_HOT"]
   },
   {
     "id": "lobster",
     "copy-from": "flesh",
     "type": "COMESTIBLE",
-    "name": { "str": "chunk of lobster meat", "str_pl": "chunks of lobster meat" },
+    "name": {
+      "str": "chunk of lobster meat",
+      "str_pl": "chunks of lobster meat"
+    },
     "description": "A chunk of uncooked lobster meat.",
     "//": "Nutrition information based on whitefish",
     "weight": "148 g",
@@ -113,9 +140,13 @@
     "price": "5 USD",
     "price_postapoc": "50 cent",
     "spoils_in": "12 hours",
-    "flags": [ "SMOKABLE", "RAW" ],
+    "flags": ["SMOKABLE", "RAW"],
     "smoking_result": "lobster_smoked",
-    "vitamins": [ [ "vitC", 2 ], [ "calcium", 6 ], [ "iron", 14 ] ],
+    "vitamins": [
+      ["vitC", 2],
+      ["calcium", 6],
+      ["iron", 14]
+    ],
     "calories": 250
   },
   {
@@ -128,7 +159,7 @@
     "spoils_in": "1 day",
     "price_postapoc": "1 USD 50 cent",
     "parasites": 0,
-    "flags": [ "EATEN_HOT" ]
+    "flags": ["EATEN_HOT"]
   },
   {
     "id": "mollusk",
@@ -142,8 +173,12 @@
     "price": "5 USD",
     "price_postapoc": "50 cent",
     "spoils_in": "12 hours",
-    "flags": [ "SMOKABLE", "RAW" ],
-    "vitamins": [ [ "vitC", 0 ], [ "calcium", 6 ], [ "iron", 2 ] ],
+    "flags": ["SMOKABLE", "RAW"],
+    "vitamins": [
+      ["vitC", 0],
+      ["calcium", 6],
+      ["iron", 2]
+    ],
     "calories": 260
   },
   {
@@ -156,7 +191,7 @@
     "spoils_in": "1 day",
     "price_postapoc": "1 USD 50 cent",
     "parasites": 0,
-    "flags": [ "EATEN_HOT" ]
+    "flags": ["EATEN_HOT"]
   },
   {
     "id": "gastropod_purged",
@@ -170,8 +205,12 @@
     "price": "5 USD",
     "price_postapoc": "50 cent",
     "spoils_in": "12 hours",
-    "flags": [ "RAW" ],
-    "vitamins": [ [ "vitC", 0 ], [ "calcium", 3 ], [ "iron", 1 ] ],
+    "flags": ["RAW"],
+    "vitamins": [
+      ["vitC", 0],
+      ["calcium", 3],
+      ["iron", 1]
+    ],
     "calories": 15
   },
   {
@@ -184,7 +223,7 @@
     "spoils_in": "1 day",
     "price_postapoc": "1 USD 50 cent",
     "parasites": 0,
-    "flags": [ "EATEN_HOT" ]
+    "flags": ["EATEN_HOT"]
   },
   {
     "id": "hstomach",
@@ -192,9 +231,12 @@
     "type": "COMESTIBLE",
     "name": { "str": "human stomach" },
     "description": "The stomach of a human.  It is surprisingly durable.",
-    "material": [ "hflesh" ],
-    "delete": { "flags": [ "PREDATOR_FUN" ] },
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "material": ["hflesh"],
+    "delete": { "flags": ["PREDATOR_FUN"] },
+    "vitamins": [
+      ["meat_allergen", 1],
+      ["human_flesh_vitamin", 1]
+    ]
   },
   {
     "id": "hstomach_large",
@@ -202,7 +244,12 @@
     "type": "COMESTIBLE",
     "name": { "str": "large human stomach" },
     "description": "The stomach of a large humanoid creature.  It is surprisingly durable.",
-    "proportional": { "weight": 2.0, "volume": 2.0, "price": 1.5, "calories": 2.0 }
+    "proportional": {
+      "weight": 2.0,
+      "volume": 2.0,
+      "price": 1.5,
+      "calories": 2.0
+    }
   },
   {
     "id": "human_fat",
@@ -210,8 +257,11 @@
     "type": "COMESTIBLE",
     "name": { "str": "chunk of human fat", "str_pl": "chunks of human fat" },
     "description": "A chunk of raw fat, harvested from a human body.",
-    "material": [ "hflesh" ],
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "material": ["hflesh"],
+    "vitamins": [
+      ["meat_allergen", 1],
+      ["human_flesh_vitamin", 1]
+    ]
   },
   {
     "type": "COMESTIBLE",
@@ -221,8 +271,11 @@
     "description": "A smooth white block of cleaned and rendered human fat.  It will remain edible for a very long time, and can be used as an ingredient in many foods and projects.",
     "price": "5 USD",
     "//": "*May* have been commercially traded.",
-    "material": [ "hflesh" ],
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "material": ["hflesh"],
+    "vitamins": [
+      ["meat_allergen", 1],
+      ["human_flesh_vitamin", 1]
+    ]
   },
   {
     "type": "COMESTIBLE",
@@ -238,9 +291,14 @@
     "cooks_like": "human_cooked",
     "name": { "str_sp": "human flesh" },
     "description": "A chunk of raw meat, butchered from a human body.",
-    "material": [ "hflesh" ],
-    "vitamins": [ [ "calcium", 2 ], [ "iron", 12 ], [ "human_flesh_vitamin", 100 ], [ "meat_allergen", 1 ] ],
-    "delete": { "flags": [ "PREDATOR_FUN" ] }
+    "material": ["hflesh"],
+    "vitamins": [
+      ["calcium", 2],
+      ["iron", 12],
+      ["human_flesh_vitamin", 1],
+      ["meat_allergen", 1]
+    ],
+    "delete": { "flags": ["PREDATOR_FUN"] }
   },
   {
     "id": "human_cooked",
@@ -251,7 +309,7 @@
     "proportional": { "price": 1.5 },
     "parasites": 0,
     "fun": 0,
-    "flags": [ "EATEN_HOT" ]
+    "flags": ["EATEN_HOT"]
   },
   {
     "id": "pepperhuman",
@@ -262,7 +320,7 @@
     "proportional": { "price": 1.5 },
     "parasites": 0,
     "fun": 2,
-    "flags": [ "EATEN_HOT" ]
+    "flags": ["EATEN_HOT"]
   },
   {
     "id": "meat",
@@ -278,8 +336,11 @@
     "calories": 402,
     "fun": -10,
     "cooks_like": "meat_cooked",
-    "vitamins": [ [ "calcium", 2 ], [ "iron", 12 ] ],
-    "flags": [ "SMOKABLE", "RAW", "PREDATOR_FUN" ],
+    "vitamins": [
+      ["calcium", 2],
+      ["iron", 12]
+    ],
+    "flags": ["SMOKABLE", "RAW", "PREDATOR_FUN"],
     "smoking_result": "meat_smoked"
   },
   {
@@ -297,7 +358,7 @@
     "symbol": "~",
     "charges": 1,
     "phase": "liquid",
-    "delete": { "flags": [ "SMOKABLE" ] }
+    "delete": { "flags": ["SMOKABLE"] }
   },
   {
     "id": "porkbelly",
@@ -312,15 +373,22 @@
     "spoils_in": "1 day",
     "calories": 1300,
     "fun": -10,
-    "vitamins": [ [ "vitC", 0 ], [ "calcium", 0 ], [ "iron", 0 ] ],
-    "flags": [ "SMOKABLE", "RAW", "PREDATOR_FUN" ],
+    "vitamins": [
+      ["vitC", 0],
+      ["calcium", 0],
+      ["iron", 0]
+    ],
+    "flags": ["SMOKABLE", "RAW", "PREDATOR_FUN"],
     "smoking_result": "meat_smoked"
   },
   {
     "id": "raw_cured_fatty_meat",
     "type": "COMESTIBLE",
     "copy-from": "flesh",
-    "name": { "str": "slab of raw cured porkbelly", "str_pl": "slabs of raw cured porkbelly" },
+    "name": {
+      "str": "slab of raw cured porkbelly",
+      "str_pl": "slabs of raw cured porkbelly"
+    },
     "description": "Porkbelly that has been preserved and flavored with salt and sugar to draw out the moisture from the first layer.  Still raw, it is ready for smoking into bacon.",
     "weight": "250 g",
     "volume": "250ml",
@@ -328,15 +396,22 @@
     "price": "6 USD 50 cent",
     "price_postapoc": "3 USD",
     "spoils_in": "14 days",
-    "vitamins": [ [ "vitC", 0 ], [ "calcium", 3 ], [ "iron", 22 ] ],
-    "flags": [ "SMOKABLE", "RAW" ],
+    "vitamins": [
+      ["vitC", 0],
+      ["calcium", 3],
+      ["iron", 22]
+    ],
+    "flags": ["SMOKABLE", "RAW"],
     "smoking_result": "bacon_uncut"
   },
   {
     "id": "bacon_uncut",
     "type": "COMESTIBLE",
     "copy-from": "meat_cooked",
-    "name": { "str": "large slab of uncut bacon", "str_pl": "large slabs of uncut bacon" },
+    "name": {
+      "str": "large slab of uncut bacon",
+      "str_pl": "large slabs of uncut bacon"
+    },
     "description": "A huge slab of uncut smoked bacon.  You can eat it as is, but it would be more efficient to cut it into smaller portions with a knife.",
     "volume": "250ml",
     "calories": 1566,
@@ -345,14 +420,21 @@
     "//": "High price to reflect the rarity and value of bacon. Note that this is three pieces of it.",
     "spoils_in": "42 days",
     "fun": 5,
-    "vitamins": [ [ "vitC", 0 ], [ "calcium", 3 ], [ "iron", 22 ] ],
-    "flags": [ "EATEN_HOT", "SMOKED" ]
+    "vitamins": [
+      ["vitC", 0],
+      ["calcium", 3],
+      ["iron", 22]
+    ],
+    "flags": ["EATEN_HOT", "SMOKED"]
   },
   {
     "id": "raw_curing_fatty_meat",
     "type": "COMESTIBLE",
     "copy-from": "flesh",
-    "name": { "str": "slab of raw curing porkbelly", "str_pl": "slabs of raw curing porkbelly" },
+    "name": {
+      "str": "slab of raw curing porkbelly",
+      "str_pl": "slabs of raw curing porkbelly"
+    },
     "description": "Porkbelly in the process of becoming fully cured with salt and sugar, capable of becoming bacon.  The flavor is slowly developing at this stage, and is significantly unpleasant due to the residual salt.  Activate it once completed to make use of the final product.",
     "weight": "275 g",
     "volume": "250ml",
@@ -361,8 +443,12 @@
     "price_postapoc": "2 USD 50 cent",
     "spoils_in": "14 days",
     "fun": -15,
-    "vitamins": [ [ "vitC", 0 ], [ "calcium", 0 ], [ "iron", 0 ] ],
-    "flags": [  ],
+    "vitamins": [
+      ["vitC", 0],
+      ["calcium", 0],
+      ["iron", 0]
+    ],
+    "flags": [],
     "use_action": {
       "target": "raw_cured_fatty_meat",
       "msg": "You pat off a bit of extra brine from the slice; this piece of meat is now fully cured.",
@@ -380,41 +466,68 @@
     "name": { "str": "scrap of meat", "str_pl": "scraps of meat" },
     "description": "A tiny scrap of raw meat.  It's not much, but it'll do in a pinch.",
     "cooks_like": "meat_scrap_cooked",
-    "proportional": { "weight": 0.1, "volume": 0.1, "price_postapoc": 0.1, "calories": 0.1 },
-    "delete": { "flags": [ "SMOKABLE" ] },
-    "vitamins": [ [ "calcium", 0 ], [ "iron", 1 ] ]
+    "proportional": {
+      "weight": 0.1,
+      "volume": 0.1,
+      "price_postapoc": 0.1,
+      "calories": 0.1
+    },
+    "delete": { "flags": ["SMOKABLE"] },
+    "vitamins": [
+      ["calcium", 0],
+      ["iron", 1]
+    ]
   },
   {
     "id": "human_meat_scrap",
     "type": "COMESTIBLE",
     "copy-from": "meat_scrap",
     "looks_like": "meat_scrap",
-    "name": { "str": "scrap of human flesh", "str_pl": "scraps of human flesh" },
+    "name": {
+      "str": "scrap of human flesh",
+      "str_pl": "scraps of human flesh"
+    },
     "description": "A tiny scrap of raw human flesh.",
-    "vitamins": [ [ "calcium", 0 ], [ "iron", 1 ], [ "human_flesh_vitamin", 10 ], [ "meat_allergen", 1 ] ],
-    "material": [ "hflesh" ]
+    "vitamins": [
+      ["calcium", 0],
+      ["iron", 1],
+      ["human_flesh_vitamin", 1],
+      ["meat_allergen", 1]
+    ],
+    "material": ["hflesh"]
   },
   {
     "id": "demihuman_meat_scrap",
     "type": "COMESTIBLE",
     "copy-from": "meat_scrap",
     "looks_like": "meat_scrap",
-    "name": { "str": "scrap of demihuman flesh", "str_pl": "scraps of demihuman flesh" },
+    "name": {
+      "str": "scrap of demihuman flesh",
+      "str_pl": "scraps of demihuman flesh"
+    },
     "description": "A tiny scrap of raw demihuman flesh.",
-    "extend": { "flags": [ "STRICT_HUMANITARIANISM" ] }
+    "extend": { "flags": ["STRICT_HUMANITARIANISM"] }
   },
   {
     "id": "mutant_meat",
     "type": "COMESTIBLE",
     "copy-from": "meat",
-    "name": { "str": "chunk of mutant meat", "str_pl": "chunks of mutant meat" },
+    "name": {
+      "str": "chunk of mutant meat",
+      "str_pl": "chunks of mutant meat"
+    },
     "snippet_category": "mutant_meat_desc",
     "description": "Raw meat from a heavily mutated animal.",
     "looks_like": "meat",
     "cooks_like": "mutant_meat_cooked",
     "proportional": { "price": 0.2, "calories": 0.5 },
-    "vitamins": [ [ "vitC", 2 ], [ "calcium", 1 ], [ "iron", 6 ], [ "mutant_toxin", 25 ] ],
-    "flags": [ "SMOKABLE", "BAD_TASTE", "RAW", "PREDATOR_FUN" ]
+    "vitamins": [
+      ["vitC", 2],
+      ["calcium", 1],
+      ["iron", 6],
+      ["mutant_toxin", 25]
+    ],
+    "flags": ["SMOKABLE", "BAD_TASTE", "RAW", "PREDATOR_FUN"]
   },
   {
     "id": "mutant_blood",
@@ -431,19 +544,22 @@
     "material": "blood",
     "charges": 1,
     "phase": "liquid",
-    "delete": { "flags": [ "SMOKABLE" ] }
+    "delete": { "flags": ["SMOKABLE"] }
   },
   {
     "id": "mutant_meat_scrap",
     "type": "COMESTIBLE",
     "copy-from": "meat_scrap",
-    "name": { "str": "scrap of mutant meat", "str_pl": "scraps of mutant meat" },
+    "name": {
+      "str": "scrap of mutant meat",
+      "str_pl": "scraps of mutant meat"
+    },
     "description": "A tiny scrap of raw meat from a heavily mutated animal.  It smells unappealing, to say the least.",
     "looks_like": "meat_scrap",
     "cooks_like": "mutant_meat_scrap_cooked",
     "proportional": { "price_postapoc": 0.2, "calories": 0.5 },
-    "vitamins": [ [ "mutant_toxin", 2 ] ],
-    "delete": { "flags": [ "SMOKABLE" ] }
+    "vitamins": [["mutant_toxin", 2]],
+    "delete": { "flags": ["SMOKABLE"] }
   },
   {
     "id": "mutant_human_flesh",
@@ -452,8 +568,15 @@
     "cooks_like": "mutant_human_cooked",
     "name": { "str_sp": "mutant humanoid meat" },
     "description": "Raw meat butchered from the body of a heavily mutated creature that was unsettlingly humanoid in appearance.  It has odd bits of fur and other tissue lodged in it that clearly don't belong.  You'd have to be crazy or starving to eat this.",
-    "material": [ "hflesh" ],
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "material": ["hflesh"],
+    "vitamins": [
+      ["vitC", 2],
+      ["calcium", 1],
+      ["iron", 6],
+      ["mutant_toxin", 25],
+      ["meat_allergen", 1],
+      ["human_flesh_vitamin", 1]
+    ]
   },
   {
     "id": "mutant_human_blood",
@@ -470,7 +593,13 @@
     "material": "hblood",
     "charges": 1,
     "phase": "liquid",
-    "delete": { "flags": [ "SMOKABLE" ] }
+    "delete": { "flags": ["SMOKABLE"] },
+    "vitamins": [
+      ["calcium", 2],
+      ["iron", 12],
+      ["meat_allergen", 1],
+      ["human_flesh_vitamin", 1]
+    ]
   },
   {
     "id": "mutant_human_cooked",
@@ -482,7 +611,7 @@
     "proportional": { "price": 1.5 },
     "parasites": 0,
     "fun": 0,
-    "flags": [ "EATEN_HOT", "NUTRIENT_OVERRIDE", "BAD_TASTE" ]
+    "flags": ["EATEN_HOT", "NUTRIENT_OVERRIDE", "BAD_TASTE"]
   },
   {
     "id": "peppermutant_human",
@@ -494,23 +623,26 @@
     "proportional": { "price": 1.5 },
     "parasites": 0,
     "fun": 1,
-    "flags": [ "EATEN_HOT", "NUTRIENT_OVERRIDE", "BAD_TASTE" ]
+    "flags": ["EATEN_HOT", "NUTRIENT_OVERRIDE", "BAD_TASTE"]
   },
   {
     "type": "COMESTIBLE",
     "id": "meat_mutant_tainted",
     "copy-from": "mutant_meat",
-    "name": { "str": "chunk of tainted mutant meat", "str_pl": "chunks of tainted mutant meat" },
+    "name": {
+      "str": "chunk of tainted mutant meat",
+      "str_pl": "chunks of tainted mutant meat"
+    },
     "color": "magenta",
     "snippet_category": "tainted_mutant_meat_desc",
     "looks_like": "meat",
     "cooks_like": "meat_mutant_tainted_cooked",
     "spoils_in": "4 hours",
-    "use_action": [ "POISON" ],
+    "use_action": ["POISON"],
     "price": "0 cent",
     "price_postapoc": "5 cent",
     "fun": -15,
-    "flags": [ "TRADER_AVOID", "SMOKABLE", "BAD_TASTE", "RAW" ],
+    "flags": ["TRADER_AVOID", "SMOKABLE", "BAD_TASTE", "RAW"],
     "smoking_result": "meat_mutant_tainted_smoked"
   },
   {
@@ -522,8 +654,8 @@
     "looks_like": "meat_cooked",
     "parasites": 0,
     "proportional": { "price_postapoc": 1.5, "fun": 0.7 },
-    "extend": { "flags": [ "EATEN_HOT" ] },
-    "delete": { "flags": [ "RAW" ] }
+    "extend": { "flags": ["EATEN_HOT"] },
+    "delete": { "flags": ["RAW"] }
   },
   {
     "type": "COMESTIBLE",
@@ -534,7 +666,7 @@
     "spoils_in": "42 days",
     "quench": -1,
     "proportional": { "price_postapoc": 1.8, "fun": 0.7 },
-    "extend": { "flags": [ "EATEN_HOT", "SMOKED" ] }
+    "extend": { "flags": ["EATEN_HOT", "SMOKED"] }
   },
   {
     "type": "COMESTIBLE",
@@ -549,12 +681,12 @@
     "color": "black",
     "looks_like": "feces_manure",
     "description": "Eugh.  This is a mess of dirt, excreta, connective tissue, and bits of matter like hair and claws, leftover from the butchering process.  Eating it isn't even worth thinking about.",
-    "material": [ "flesh" ],
+    "material": ["flesh"],
     "spoils_in": "8 hours",
     "fun": -18,
-    "use_action": [ "POISON" ],
-    "contamination": [ { "disease": "bad_food", "probability": 100 } ],
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "use_action": ["POISON"],
+    "contamination": [{ "disease": "bad_food", "probability": 100 }],
+    "vitamins": [["meat_allergen", 1]]
   },
   {
     "id": "meat_cooked",
@@ -569,8 +701,11 @@
     "price_postapoc": "1 USD",
     "parasites": 0,
     "calories": 402,
-    "vitamins": [ [ "calcium", 2 ], [ "iron", 12 ] ],
-    "flags": [ "EATEN_HOT", "NUTRIENT_OVERRIDE" ]
+    "vitamins": [
+      ["calcium", 2],
+      ["iron", 12]
+    ],
+    "flags": ["EATEN_HOT", "NUTRIENT_OVERRIDE"]
   },
   {
     "id": "peppermeat",
@@ -586,8 +721,11 @@
     "fun": 2,
     "parasites": 0,
     "calories": 402,
-    "vitamins": [ [ "calcium", 2 ], [ "iron", 12 ] ],
-    "flags": [ "EATEN_HOT", "NUTRIENT_OVERRIDE" ]
+    "vitamins": [
+      ["calcium", 2],
+      ["iron", 12]
+    ],
+    "flags": ["EATEN_HOT", "NUTRIENT_OVERRIDE"]
   },
   {
     "id": "meat_fatty_cooked",
@@ -604,8 +742,8 @@
     "//": "Luxury good.",
     "fun": 2,
     "parasites": 0,
-    "vitamins": [  ],
-    "flags": [ "EATEN_HOT" ]
+    "vitamins": [],
+    "flags": ["EATEN_HOT"]
   },
   {
     "id": "pepperfat",
@@ -621,28 +759,34 @@
     "price_postapoc": "3 USD 20 cent",
     "fun": 3,
     "parasites": 0,
-    "vitamins": [  ],
-    "flags": [ "EATEN_HOT" ]
+    "vitamins": [],
+    "flags": ["EATEN_HOT"]
   },
   {
     "id": "meat_scrap_cooked",
     "copy-from": "meat_scrap",
     "type": "COMESTIBLE",
-    "name": { "str": "cooked scrap of meat", "str_pl": "cooked scraps of meat" },
+    "name": {
+      "str": "cooked scrap of meat",
+      "str_pl": "cooked scraps of meat"
+    },
     "parasites": 0,
     "fun": 0,
-    "flags": [ "NUTRIENT_OVERRIDE" ],
+    "flags": ["NUTRIENT_OVERRIDE"],
     "calories": 51
   },
   {
     "id": "pepperscrap",
     "copy-from": "meat_scrap",
     "type": "COMESTIBLE",
-    "name": { "str": "cooked pepper scrap of meat", "str_pl": "cooked pepper scraps of meat" },
+    "name": {
+      "str": "cooked pepper scrap of meat",
+      "str_pl": "cooked pepper scraps of meat"
+    },
     "description": "A cooked scrap of meat that has been properly seasoned with pepper and salt.  Yet small, they have a good taste.",
     "parasites": 0,
     "fun": 1,
-    "flags": [ "NUTRIENT_OVERRIDE" ],
+    "flags": ["NUTRIENT_OVERRIDE"],
     "calories": 51
   },
   {
@@ -656,7 +800,7 @@
     "proportional": { "price": 1.5 },
     "parasites": 0,
     "fun": -1,
-    "flags": [ "EATEN_HOT", "NUTRIENT_OVERRIDE", "BAD_TASTE" ]
+    "flags": ["EATEN_HOT", "NUTRIENT_OVERRIDE", "BAD_TASTE"]
   },
   {
     "id": "peppermutant",
@@ -669,38 +813,50 @@
     "proportional": { "price": 1.5 },
     "parasites": 0,
     "fun": 0,
-    "flags": [ "EATEN_HOT", "NUTRIENT_OVERRIDE", "BAD_TASTE" ]
+    "flags": ["EATEN_HOT", "NUTRIENT_OVERRIDE", "BAD_TASTE"]
   },
   {
     "id": "mutant_meat_scrap_cooked",
     "type": "COMESTIBLE",
     "copy-from": "mutant_meat_scrap",
-    "name": { "str": "cooked scrap of mutant meat", "str_pl": "cooked scraps of mutant meat" },
+    "name": {
+      "str": "cooked scrap of mutant meat",
+      "str_pl": "cooked scraps of mutant meat"
+    },
     "description": "A tiny scrap of cooked mutant meat.  It is small enough that it's hard to tell how disgusting it is.",
     "looks_like": "meat_scrap_cooked",
     "parasites": 0,
     "calories": 25,
     "fun": 0,
-    "flags": [ "NUTRIENT_OVERRIDE" ]
+    "flags": ["NUTRIENT_OVERRIDE"]
   },
   {
     "id": "pepperscrap_mutant",
     "type": "COMESTIBLE",
     "copy-from": "mutant_meat_scrap",
-    "name": { "str": "cooked pepper scrap of mutant meat", "str_pl": "cooked pepper scraps of mutant meat" },
+    "name": {
+      "str": "cooked pepper scrap of mutant meat",
+      "str_pl": "cooked pepper scraps of mutant meat"
+    },
     "description": "A tiny scrap of cooked mutant meat properly seasoned with black pepper and salt.  It is small, and even though it has been seasoned, it has a weird taste.",
     "looks_like": "meat_scrap_cooked",
     "parasites": 0,
     "calories": 25,
     "fun": 1,
-    "flags": [ "NUTRIENT_OVERRIDE" ]
+    "flags": ["NUTRIENT_OVERRIDE"]
   },
   {
     "id": "cooked_marrow",
     "type": "COMESTIBLE",
     "comestible_type": "FOOD",
     "name": { "str_sp": "roasted bone marrow" },
-    "conditional_names": [ { "type": "VITAMIN", "condition": "human_flesh_vitamin", "name": { "str_sp": "Marvin marrow" } } ],
+    "conditional_names": [
+      {
+        "type": "VITAMIN",
+        "condition": "human_flesh_vitamin",
+        "name": { "str_sp": "Marvin marrow" }
+      }
+    ],
     "description": "Bone marrow, fully cooked to eliminate any parasites.  Tasty!",
     "symbol": "%",
     "color": "red",
@@ -708,18 +864,27 @@
     "volume": "50 ml",
     "price": "5 USD",
     "price_postapoc": "50 cent",
-    "material": [ "flesh" ],
+    "material": ["flesh"],
     "spoils_in": "1 day",
     "calories": 471,
     "fun": 5,
-    "vitamins": [ [ "iron", 17 ], [ "meat_allergen", 1 ] ]
+    "vitamins": [
+      ["iron", 17],
+      ["meat_allergen", 1]
+    ]
   },
   {
     "id": "pepperbone",
     "type": "COMESTIBLE",
     "comestible_type": "FOOD",
     "name": { "str_sp": "roasted pepper bone marrow" },
-    "conditional_names": [ { "type": "VITAMIN", "condition": "human_flesh_vitamin", "name": { "str_sp": "Marvin marrow" } } ],
+    "conditional_names": [
+      {
+        "type": "VITAMIN",
+        "condition": "human_flesh_vitamin",
+        "name": { "str_sp": "Marvin marrow" }
+      }
+    ],
     "description": "Bone marrow, fully cooked to eliminate any parasites and properly seasoned with black pepper and salt.  Tasty!",
     "symbol": "%",
     "color": "red",
@@ -727,11 +892,14 @@
     "volume": "50 ml",
     "price": "5 USD",
     "price_postapoc": "50 cent",
-    "material": [ "flesh" ],
+    "material": ["flesh"],
     "spoils_in": "1 day",
     "calories": 471,
     "fun": 6,
-    "vitamins": [ [ "iron", 17 ], [ "meat_allergen", 1 ] ]
+    "vitamins": [
+      ["iron", 17],
+      ["meat_allergen", 1]
+    ]
   },
   {
     "id": "offal",
@@ -741,8 +909,8 @@
     "description": "A selection of uncooked internal organs and entrails.  Filled with essential vitamins, but most people consider it a bit gross unless it's very carefully prepared.",
     "proportional": { "parasites": 0.5 },
     "price_postapoc": "25 cent",
-    "delete": { "flags": [ "SMOKABLE" ] },
-    "relative": { "vitamins": [ [ "iron", 4 ] ], "fun": -5 }
+    "delete": { "flags": ["SMOKABLE"] },
+    "relative": { "vitamins": [["iron", 4]], "fun": -5 }
   },
   {
     "id": "offal_cooked",
@@ -753,7 +921,7 @@
     "parasites": 0,
     "price_postapoc": "50 cent",
     "fun": -5,
-    "flags": [ "EATEN_HOT" ]
+    "flags": ["EATEN_HOT"]
   },
   {
     "id": "tainted_innards",
@@ -763,7 +931,7 @@
     "description": "An unidentifiable selection of rotted innards that barely hold their form.  It is revolting and certainly no longer healthy to eat.",
     "snippet_category": "tainted_innards_desc",
     "spoils_in": "4 hours",
-    "use_action": [ "POISON" ],
+    "use_action": ["POISON"],
     "price_postapoc": "0 cent",
     "fun": -25
   },
@@ -777,8 +945,8 @@
     "looks_like": "offal",
     "cooks_like": "mutant_bug_organs_cooked",
     "proportional": { "price": 0.1, "calories": 0.65 },
-    "vitamins": [ [ "mutant_toxin", 25 ] ],
-    "extend": { "flags": [ "BAD_TASTE" ] }
+    "vitamins": [["mutant_toxin", 25]],
+    "extend": { "flags": ["BAD_TASTE"] }
   },
   {
     "id": "mutant_bug_organs_cooked",
@@ -789,7 +957,7 @@
     "description": "A selection of cooked mutant bug organs.  Looks awful, but if you give it a try, you'll find -- that it's worse than it looks.",
     "looks_like": "offal_cooked",
     "parasites": 0,
-    "delete": { "flags": [ "RAW", "NUTRIENT_OVERRIDE" ] }
+    "delete": { "flags": ["RAW", "NUTRIENT_OVERRIDE"] }
   },
   {
     "id": "offal_pickled",
@@ -804,7 +972,7 @@
     "spoils_in": "1 day 12 hours",
     "quench": -1,
     "fun": -1,
-    "delete": { "flags": [ "RAW" ] }
+    "delete": { "flags": ["RAW"] }
   },
   {
     "id": "offal_canned",
@@ -817,7 +985,7 @@
     "parasites": 0,
     "container": "can_medium",
     "fun": -2,
-    "delete": { "flags": [ "RAW" ] }
+    "delete": { "flags": ["RAW"] }
   },
   {
     "id": "stomach",
@@ -833,7 +1001,7 @@
     "calories": 60,
     "quench": 1,
     "fun": -15,
-    "extend": { "flags": [ "PREDATOR_FUN" ] }
+    "extend": { "flags": ["PREDATOR_FUN"] }
   },
   {
     "id": "stomach_large",
@@ -841,7 +1009,12 @@
     "type": "COMESTIBLE",
     "name": { "str": "large stomach" },
     "description": "The stomach of a large woodland creature.  It is surprisingly durable.",
-    "proportional": { "weight": 2.0, "volume": 2.0, "price": 1.5, "calories": 2.0 }
+    "proportional": {
+      "weight": 2.0,
+      "volume": 2.0,
+      "price": 1.5,
+      "calories": 2.0
+    }
   },
   {
     "id": "jerky",
@@ -849,9 +1022,21 @@
     "type": "COMESTIBLE",
     "name": { "str_sp": "meat jerky" },
     "conditional_names": [
-      { "type": "VITAMIN", "condition": "human_flesh_vitamin", "name": { "str_sp": "jerk jerky" } },
-      { "type": "FLAG", "condition": "STRICT_HUMANITARIANISM", "name": { "str_sp": "talking animal jerky" } },
-      { "type": "COMPONENT_ID_SUBSTRING", "condition": "mutant", "name": { "str_sp": "monster jerky" } }
+      {
+        "type": "VITAMIN",
+        "condition": "human_flesh_vitamin",
+        "name": { "str_sp": "jerk jerky" }
+      },
+      {
+        "type": "FLAG",
+        "condition": "STRICT_HUMANITARIANISM",
+        "name": { "str_sp": "talking animal jerky" }
+      },
+      {
+        "type": "COMPONENT_ID_SUBSTRING",
+        "condition": "mutant",
+        "name": { "str_sp": "monster jerky" }
+      }
     ],
     "description": "Salty dried meat that lasts for a long time, but will make you thirsty.",
     "primary_material": "cured_meat",
@@ -866,8 +1051,12 @@
     "quench": -5,
     "fun": 4,
     "calories": 348,
-    "vitamins": [ [ "vitC", 0 ], [ "calcium", 2 ], [ "iron", 26 ] ],
-    "delete": { "flags": [ "RAW" ] }
+    "vitamins": [
+      ["vitC", 0],
+      ["calcium", 2],
+      ["iron", 26]
+    ],
+    "delete": { "flags": ["RAW"] }
   },
   {
     "id": "salted_fish",
@@ -885,9 +1074,13 @@
     "fun": 4,
     "volume": "50 ml",
     "proportional": { "price": 2.0, "weight": 0.345 },
-    "vitamins": [ [ "vitC", 1 ], [ "calcium", 3 ], [ "iron", 7 ] ],
+    "vitamins": [
+      ["vitC", 1],
+      ["calcium", 3],
+      ["iron", 7]
+    ],
     "calories": 130,
-    "delete": { "flags": [ "RAW" ] }
+    "delete": { "flags": ["RAW"] }
   },
   {
     "id": "salted_mollusk",
@@ -904,7 +1097,7 @@
     "quench": -5,
     "fun": 4,
     "proportional": { "price": 2.0, "weight": 0.345 },
-    "delete": { "flags": [ "RAW" ] }
+    "delete": { "flags": ["RAW"] }
   },
   {
     "id": "meat_smoked",
@@ -912,9 +1105,21 @@
     "type": "COMESTIBLE",
     "name": { "str_sp": "smoked meat" },
     "conditional_names": [
-      { "type": "VITAMIN", "condition": "human_flesh_vitamin", "name": { "str_sp": "smoked sucker" } },
-      { "type": "FLAG", "condition": "STRICT_HUMANITARIANISM", "name": { "str_sp": "smoked Narnian" } },
-      { "type": "COMPONENT_ID_SUBSTRING", "condition": "mutant", "name": { "str_sp": "%s, mutant" } }
+      {
+        "type": "VITAMIN",
+        "condition": "human_flesh_vitamin",
+        "name": { "str_sp": "smoked sucker" }
+      },
+      {
+        "type": "FLAG",
+        "condition": "STRICT_HUMANITARIANISM",
+        "name": { "str_sp": "smoked Narnian" }
+      },
+      {
+        "type": "COMPONENT_ID_SUBSTRING",
+        "condition": "mutant",
+        "name": { "str_sp": "%s, mutant" }
+      }
     ],
     "description": "Tasty meat that has been heavily smoked for preservation.  It could be further smoked to dehydrate it completely.",
     "color": "brown",
@@ -922,7 +1127,7 @@
     "price_postapoc": "2 USD",
     "quench": -1,
     "fun": 5,
-    "flags": [ "EATEN_HOT", "SMOKED", "SMOKABLE" ],
+    "flags": ["EATEN_HOT", "SMOKED", "SMOKABLE"],
     "smoking_result": "dry_meat"
   },
   {
@@ -935,7 +1140,7 @@
     "spoils_in": "42 days",
     "price_postapoc": "2 USD",
     "fun": 6,
-    "flags": [ "EATEN_HOT", "SMOKED", "SMOKABLE" ],
+    "flags": ["EATEN_HOT", "SMOKED", "SMOKABLE"],
     "smoking_result": "dry_fish"
   },
   {
@@ -947,7 +1152,7 @@
     "color": "brown",
     "price_postapoc": "3 USD",
     "fun": 10,
-    "flags": [ "EATEN_HOT", "SMOKED", "SMOKABLE" ],
+    "flags": ["EATEN_HOT", "SMOKED", "SMOKABLE"],
     "smoking_result": "dry_lobster"
   },
   {
@@ -959,7 +1164,7 @@
     "color": "brown",
     "price_postapoc": "3 USD",
     "fun": 10,
-    "flags": [ "EATEN_HOT", "SMOKED", "SMOKABLE" ],
+    "flags": ["EATEN_HOT", "SMOKED", "SMOKABLE"],
     "smoking_result": "dry_mollusk"
   },
   {
@@ -977,27 +1182,43 @@
     "quench": 2,
     "fun": -10,
     "calories": 40,
-    "vitamins": [ [ "vitC", 9 ], [ "calcium", 0 ], [ "iron", 14 ] ],
-    "extend": { "flags": [ "PREDATOR_FUN" ] }
+    "vitamins": [
+      ["vitC", 9],
+      ["calcium", 0],
+      ["iron", 14]
+    ],
+    "extend": { "flags": ["PREDATOR_FUN"] }
   },
   {
     "id": "human_lung",
     "type": "COMESTIBLE",
     "copy-from": "lung",
     "looks_like": "lung",
-    "name": { "str": "piece of raw human lung", "str_pl": "pieces of raw human lung" },
+    "name": {
+      "str": "piece of raw human lung",
+      "str_pl": "pieces of raw human lung"
+    },
     "description": "A portion of lung from a human.  It's spongy and pink, and spoils very quickly.  It appears to have been extracted too crudely to ever be useful as a transplant.",
-    "material": [ "hflesh" ],
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "material": ["hflesh"],
+    "vitamins": [
+      ["vitC", 9],
+      ["calcium", 0],
+      ["iron", 14],
+      ["meat_allergen", 1],
+      ["human_flesh_vitamin", 1]
+    ]
   },
   {
     "id": "demihuman_lung",
     "type": "COMESTIBLE",
     "copy-from": "lung",
     "looks_like": "lung",
-    "name": { "str": "piece of raw demihuman lung", "str_pl": "pieces of raw demihuman lung" },
+    "name": {
+      "str": "piece of raw demihuman lung",
+      "str_pl": "pieces of raw demihuman lung"
+    },
     "description": "A portion of lung from a demihuman being.  It's spongy and pink, and spoils very quickly.",
-    "extend": { "flags": [ "STRICT_HUMANITARIANISM" ] }
+    "extend": { "flags": ["STRICT_HUMANITARIANISM"] }
   },
   {
     "id": "lung_cooked",
@@ -1005,42 +1226,66 @@
     "copy-from": "lung",
     "fun": -5,
     "looks_like": "offal_cooked",
-    "name": { "str": "cooked piece of lung", "str_pl": "cooked pieces of lung" },
+    "name": {
+      "str": "cooked piece of lung",
+      "str_pl": "cooked pieces of lung"
+    },
     "description": "A chewy, grayish lump of flavorless connective tissue.  It doesn't look any tastier than it did raw, but any parasites are all cooked out.",
     "parasites": 0,
-    "extend": { "flags": [ "EATEN_HOT" ] },
-    "delete": { "flags": [ "RAW" ] }
+    "extend": { "flags": ["EATEN_HOT"] },
+    "delete": { "flags": ["RAW"] }
   },
   {
     "id": "mutant_lung",
     "type": "COMESTIBLE",
     "copy-from": "lung",
-    "name": { "str": "piece of raw mutant lung", "str_pl": "pieces of raw mutant lung" },
+    "name": {
+      "str": "piece of raw mutant lung",
+      "str_pl": "pieces of raw mutant lung"
+    },
     "snippet_category": "mutant_lung_desc",
     "fun": -15,
     "proportional": { "price_postapoc": 0.5 },
-    "extend": { "flags": [ "BAD_TASTE" ] },
-    "vitamins": [ [ "mutant_toxin", 10 ] ]
+    "extend": { "flags": ["BAD_TASTE"] },
+    "vitamins": [
+      ["vitC", 9],
+      ["calcium", 0],
+      ["iron", 14],
+      ["mutant_toxin", 10],
+      ["meat_allergen", 1]
+    ]
   },
   {
     "id": "mutant_lung_cooked",
     "type": "COMESTIBLE",
     "copy-from": "lung_cooked",
-    "name": { "str": "cooked piece of mutant lung", "str_pl": "cooked pieces of mutant lung" },
+    "name": {
+      "str": "cooked piece of mutant lung",
+      "str_pl": "cooked pieces of mutant lung"
+    },
     "snippet_category": "cooked_mutant_lung_desc",
-    "delete": { "flags": [ "RAW" ] }
+    "delete": { "flags": ["RAW"] }
   },
   {
     "id": "mutant_bug_lungs",
     "type": "COMESTIBLE",
     "copy-from": "lung",
-    "name": { "str": "piece of raw mutant lung", "str_pl": "pieces of raw mutant lung" },
+    "name": {
+      "str": "piece of raw mutant lung",
+      "str_pl": "pieces of raw mutant lung"
+    },
     "snippet_category": "mutant_bug_lung_desc",
     "description": "You're pretty sure this is lung tissue.",
     "looks_like": "lung",
     "proportional": { "price_postapoc": 0.1, "calories": 0.65 },
-    "vitamins": [ [ "mutant_toxin", 25 ] ],
-    "extend": { "flags": [ "BAD_TASTE" ] }
+    "vitamins": [
+      ["vitC", 9],
+      ["calcium", 0],
+      ["iron", 14],
+      ["mutant_toxin", 25],
+      ["meat_allergen", 1]
+    ],
+    "extend": { "flags": ["BAD_TASTE"] }
   },
   {
     "id": "liver",
@@ -1052,55 +1297,86 @@
     "price_postapoc": "25 cent",
     "quench": -4,
     "calories": 92,
-    "vitamins": [ [ "vitC", 0 ], [ "calcium", 1 ], [ "iron", 28 ] ]
+    "vitamins": [
+      ["vitC", 0],
+      ["calcium", 1],
+      ["iron", 28]
+    ]
   },
   {
     "id": "human_liver",
     "type": "COMESTIBLE",
     "copy-from": "liver",
     "looks_like": "liver",
-    "name": { "str": "piece of raw human liver", "str_pl": "pieces of raw human liver" },
+    "name": {
+      "str": "piece of raw human liver",
+      "str_pl": "pieces of raw human liver"
+    },
     "description": "A piece of liver from a human, which appears to have been extracted too crudely to ever be useful as a transplant.",
-    "material": [ "hflesh" ],
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "material": ["hflesh"],
+    "vitamins": [
+      ["vitC", 0],
+      ["calcium", 1],
+      ["iron", 28],
+      ["meat_allergen", 1],
+      ["human_flesh_vitamin", 1]
+    ]
   },
   {
     "id": "demihuman_liver",
     "type": "COMESTIBLE",
     "copy-from": "liver",
     "looks_like": "liver",
-    "name": { "str": "piece of raw demihuman liver", "str_pl": "pieces of raw demihuman liver" },
+    "name": {
+      "str": "piece of raw demihuman liver",
+      "str_pl": "pieces of raw demihuman liver"
+    },
     "description": "A chunk of liver from a demihuman being.",
-    "extend": { "flags": [ "STRICT_HUMANITARIANISM" ] }
+    "extend": { "flags": ["STRICT_HUMANITARIANISM"] }
   },
   {
     "id": "liver_cooked",
     "type": "COMESTIBLE",
     "copy-from": "liver",
     "looks_like": "meat_cooked",
-    "name": { "str": "cooked piece of liver", "str_pl": "cooked pieces of liver" },
+    "name": {
+      "str": "cooked piece of liver",
+      "str_pl": "cooked pieces of liver"
+    },
     "description": "Chock-full of A- and B-Vitamins!  Cooked liver isn't all that bad, depending on how you feel about the texture, but this is probably the least fancy way to do it.",
     "parasites": 0,
     "fun": 0,
     "price_postapoc": "50 cent",
-    "delete": { "flags": [ "RAW" ] }
+    "delete": { "flags": ["RAW"] }
   },
   {
     "id": "mutant_liver",
     "type": "COMESTIBLE",
     "copy-from": "liver",
-    "name": { "str": "piece of raw mutant liver", "str_pl": "pieces of raw mutant liver" },
+    "name": {
+      "str": "piece of raw mutant liver",
+      "str_pl": "pieces of raw mutant liver"
+    },
     "snippet_category": "mutant_liver_desc",
     "fun": -15,
     "proportional": { "price_postapoc": 0.5 },
-    "extend": { "flags": [ "BAD_TASTE" ] },
-    "vitamins": [ [ "mutant_toxin", 40 ] ]
+    "extend": { "flags": ["BAD_TASTE"] },
+    "vitamins": [
+      ["vitC", 0],
+      ["calcium", 1],
+      ["iron", 28],
+      ["mutant_toxin", 40],
+      ["meat_allergen", 1]
+    ]
   },
   {
     "id": "mutant_liver_cooked",
     "type": "COMESTIBLE",
     "copy-from": "liver_cooked",
-    "name": { "str": "cooked piece of mutant liver", "str_pl": "cooked pieces of mutant liver" },
+    "name": {
+      "str": "cooked piece of mutant liver",
+      "str_pl": "cooked pieces of mutant liver"
+    },
     "snippet_category": "cooked_mutant_liver_desc",
     "fun": -5
   },
@@ -1108,7 +1384,10 @@
     "id": "bone_marrow",
     "copy-from": "flesh",
     "type": "COMESTIBLE",
-    "name": { "str": "portion of raw bone marrow", "str_pl": "portions of raw bone marrow" },
+    "name": {
+      "str": "portion of raw bone marrow",
+      "str_pl": "portions of raw bone marrow"
+    },
     "description": "A soft, spongy substance found in bones, rich in fat.  Chock-full of good stuff and doesn't taste too bad even raw, but beware of parasites.",
     "weight": "60 g",
     "volume": "50 ml",
@@ -1117,44 +1396,67 @@
     "spoils_in": "1 day",
     "calories": 471,
     "fun": -3,
-    "vitamins": [ [ "iron", 17 ] ],
-    "flags": [ "RAW", "PREDATOR_FUN" ]
+    "vitamins": [["iron", 17]],
+    "flags": ["RAW", "PREDATOR_FUN"]
   },
   {
     "id": "human_marrow",
     "type": "COMESTIBLE",
     "copy-from": "bone_marrow",
-    "name": { "str": "portion of raw human bone marrow", "str_pl": "portions of raw human bone marrow" },
+    "name": {
+      "str": "portion of raw human bone marrow",
+      "str_pl": "portions of raw human bone marrow"
+    },
     "description": "Soft, spongy marrow tissue from a human bone.",
-    "material": [ "hflesh" ],
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "material": ["hflesh"],
+    "vitamins": [
+      ["iron", 17],
+      ["meat_allergen", 1],
+      ["human_flesh_vitamin", 1]
+    ]
   },
   {
     "id": "demihuman_marrow",
     "type": "COMESTIBLE",
     "copy-from": "bone_marrow",
-    "name": { "str": "portion of raw demihuman bone marrow", "str_pl": "portions of raw demihuman bone marrow" },
+    "name": {
+      "str": "portion of raw demihuman bone marrow",
+      "str_pl": "portions of raw demihuman bone marrow"
+    },
     "description": "Soft, spongy marrow tissue from a demihuman bone.",
-    "extend": { "flags": [ "STRICT_HUMANITARIANISM" ] }
+    "extend": { "flags": ["STRICT_HUMANITARIANISM"] }
   },
   {
     "id": "mutant_marrow",
     "type": "COMESTIBLE",
     "copy-from": "bone_marrow",
-    "name": { "str": "portion of raw mutant bone marrow", "str_pl": "portions of raw mutant bone marrow" },
+    "name": {
+      "str": "portion of raw mutant bone marrow",
+      "str_pl": "portions of raw mutant bone marrow"
+    },
     "description": "Soft, spongy and oddly-colored marrow tissue from the bone of a heavily mutated animal.",
     "proportional": { "price": 0.2, "calories": 0.5 },
-    "vitamins": [ [ "iron", 7 ], [ "mutant_toxin", 25 ] ],
-    "flags": [ "BAD_TASTE", "RAW", "PREDATOR_FUN" ]
+    "vitamins": [
+      ["iron", 7],
+      ["mutant_toxin", 25]
+    ],
+    "flags": ["BAD_TASTE", "RAW", "PREDATOR_FUN"]
   },
   {
     "id": "mutant_human_marrow",
     "type": "COMESTIBLE",
     "copy-from": "mutant_marrow",
-    "name": { "str": "portion of raw mutant human bone marrow", "str_pl": "portions of raw mutant human bone marrow" },
+    "name": {
+      "str": "portion of raw mutant human bone marrow",
+      "str_pl": "portions of raw mutant human bone marrow"
+    },
     "description": "Soft, spongy and oddly-colored marrow tissue from the bone of a heavily mutated creature that was unsettlingly humanoid in appearance.",
-    "material": [ "hflesh" ],
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "material": ["hflesh"],
+    "vitamins": [
+      ["iron", 17],
+      ["meat_allergen", 1],
+      ["human_flesh_vitamin", 1]
+    ]
   },
   {
     "id": "brain",
@@ -1168,60 +1470,90 @@
     "quench": 10,
     "parasites": 15,
     "calories": 78,
-    "vitamins": [ [ "vitC", 3 ], [ "calcium", 0 ], [ "iron", 10 ] ]
+    "vitamins": [
+      ["vitC", 3],
+      ["calcium", 0],
+      ["iron", 10]
+    ]
   },
   {
     "id": "human_brain",
     "type": "COMESTIBLE",
     "copy-from": "brain",
     "looks_like": "brain",
-    "name": { "str": "piece of raw human brain", "str_pl": "pieces of raw human brain" },
+    "name": {
+      "str": "piece of raw human brain",
+      "str_pl": "pieces of raw human brain"
+    },
     "description": "A chunk of brain from a human.  Eating this may cause kuru, but you probably won't live long enough to care about that anyway.",
-    "material": [ "hflesh" ],
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "material": ["hflesh"],
+    "vitamins": [
+      ["vitC", 3],
+      ["calcium", 0],
+      ["iron", 10],
+      ["meat_allergen", 1],
+      ["human_flesh_vitamin", 1]
+    ]
   },
   {
     "id": "demihuman_brain",
     "type": "COMESTIBLE",
     "copy-from": "brain",
     "looks_like": "brain",
-    "name": { "str": "piece of raw demihuman brain", "str_pl": "pieces of raw demihuman brain" },
+    "name": {
+      "str": "piece of raw demihuman brain",
+      "str_pl": "pieces of raw demihuman brain"
+    },
     "description": "A chunk of brain from a demihuman being.  Don't prion diseases cross species barriers?",
-    "extend": { "flags": [ "STRICT_HUMANITARIANISM" ] }
+    "extend": { "flags": ["STRICT_HUMANITARIANISM"] }
   },
   {
     "id": "brain_cooked",
     "type": "COMESTIBLE",
     "copy-from": "brain",
     "looks_like": "offal_cooked",
-    "name": { "str": "cooked piece of brain", "str_pl": "cooked pieces of brain" },
+    "name": {
+      "str": "cooked piece of brain",
+      "str_pl": "cooked pieces of brain"
+    },
     "description": "Now you can emulate those zombies you love so much!  Preparing brain for eating is challenging, and this doesn't seem to be the best way to do it.",
     "price_postapoc": "25 cent",
     "parasites": 0,
     "fun": -5,
-    "extend": { "flags": [ "EATEN_HOT" ] },
-    "delete": { "flags": [ "RAW" ] }
+    "extend": { "flags": ["EATEN_HOT"] },
+    "delete": { "flags": ["RAW"] }
   },
   {
     "id": "mutant_brain",
     "type": "COMESTIBLE",
     "copy-from": "brain",
     "looks_like": "brain",
-    "name": { "str": "piece of raw mutant brain", "str_pl": "pieces of raw mutant brain" },
+    "name": {
+      "str": "piece of raw mutant brain",
+      "str_pl": "pieces of raw mutant brain"
+    },
     "snippet_category": "mutant_brain_desc",
     "proportional": { "price_postapoc": 0.5 },
     "fun": -20,
-    "extend": { "flags": [ "BAD_TASTE" ] },
-    "vitamins": [ [ "mutant_toxin", 50 ] ]
+    "extend": { "flags": ["BAD_TASTE"] },
+    "vitamins": [
+      ["vitC", 3],
+      ["iron", 10],
+      ["mutant_toxin", 50],
+      ["meat_allergen", 1]
+    ]
   },
   {
     "id": "mutant_brain_cooked",
     "type": "COMESTIBLE",
     "copy-from": "brain_cooked",
-    "name": { "str": "cooked piece of mutant brain", "str_pl": "cooked pieces of mutant brain" },
+    "name": {
+      "str": "cooked piece of mutant brain",
+      "str_pl": "cooked pieces of mutant brain"
+    },
     "snippet_category": "cooked_mutant_brain_desc",
     "fun": -12,
-    "delete": { "flags": [ "RAW" ] }
+    "delete": { "flags": ["RAW"] }
   },
   {
     "id": "kidney",
@@ -1233,103 +1565,156 @@
     "color": "brown",
     "price_postapoc": "25 cent",
     "calories": 54,
-    "vitamins": [ [ "vitC", 0 ], [ "calcium", 1 ], [ "iron", 25 ] ]
+    "vitamins": [
+      ["vitC", 0],
+      ["calcium", 1],
+      ["iron", 25]
+    ]
   },
   {
     "id": "human_kidney",
     "type": "COMESTIBLE",
     "copy-from": "kidney",
     "looks_like": "kidney",
-    "name": { "str": "piece of raw human kidney", "str_pl": "pieces of raw human kidney" },
+    "name": {
+      "str": "piece of raw human kidney",
+      "str_pl": "pieces of raw human kidney"
+    },
     "description": "A chunk of kidney from a human, which appears to have been extracted too crudely to ever be useful as a transplant.",
-    "material": [ "hflesh" ],
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "material": ["hflesh"],
+    "vitamins": [
+      ["vitC", 0],
+      ["calcium", 1],
+      ["iron", 25],
+      ["meat_allergen", 1],
+      ["human_flesh_vitamin", 1]
+    ]
   },
   {
     "id": "demihuman_kidney",
     "type": "COMESTIBLE",
     "copy-from": "kidney",
     "looks_like": "kidney",
-    "name": { "str": "piece of raw demihuman kidney", "str_pl": "pieces of raw demihuman kidney" },
+    "name": {
+      "str": "piece of raw demihuman kidney",
+      "str_pl": "pieces of raw demihuman kidney"
+    },
     "description": "This is probably a piece of kidney extracted from a demihuman being.",
-    "extend": { "flags": [ "STRICT_HUMANITARIANISM" ] }
+    "extend": { "flags": ["STRICT_HUMANITARIANISM"] }
   },
   {
     "id": "kidney_cooked",
     "type": "COMESTIBLE",
     "copy-from": "kidney",
     "looks_like": "offal_cooked",
-    "name": { "str": "cooked piece of kidney", "str_pl": "cooked pieces of kidney" },
+    "name": {
+      "str": "cooked piece of kidney",
+      "str_pl": "cooked pieces of kidney"
+    },
     "description": "No, this is not beans.",
     "price_postapoc": "25 cent",
     "quench": 0,
     "fun": -2,
     "parasites": 0,
-    "delete": { "flags": [ "RAW" ] }
+    "delete": { "flags": ["RAW"] }
   },
   {
     "id": "mutant_kidney",
     "type": "COMESTIBLE",
     "copy-from": "kidney",
     "fun": -15,
-    "name": { "str": "piece of raw mutant kidney", "str_pl": "pieces of raw mutant kidney" },
+    "name": {
+      "str": "piece of raw mutant kidney",
+      "str_pl": "pieces of raw mutant kidney"
+    },
     "snippet_category": "mutant_kidney_desc",
     "proportional": { "price_postapoc": 0.5 },
-    "extend": { "flags": [ "BAD_TASTE" ] },
-    "vitamins": [ [ "mutant_toxin", 25 ] ]
+    "extend": { "flags": ["BAD_TASTE"] },
+    "vitamins": [
+      ["vitC", 0],
+      ["calcium", 1],
+      ["iron", 25],
+      ["mutant_toxin", 25],
+      ["meat_allergen", 1]
+    ]
   },
   {
     "id": "mutant_kidney_cooked",
     "type": "COMESTIBLE",
     "copy-from": "kidney_cooked",
-    "name": { "str": "cooked piece of mutant kidney", "str_pl": "cooked pieces of mutant kidney" },
+    "name": {
+      "str": "cooked piece of mutant kidney",
+      "str_pl": "cooked pieces of mutant kidney"
+    },
     "snippet_category": "cooked_mutant_kidney_desc",
     "fun": -12,
-    "delete": { "flags": [ "RAW" ] }
+    "delete": { "flags": ["RAW"] }
   },
   {
     "id": "sweetbread",
     "type": "COMESTIBLE",
     "copy-from": "lung",
     "looks_like": "offal",
-    "name": { "str": "piece of raw sweetbread", "str_pl": "pieces of raw sweetbread" },
+    "name": {
+      "str": "piece of raw sweetbread",
+      "str_pl": "pieces of raw sweetbread"
+    },
     "description": "A piece of the thymus or pancreas of an animal.  These are a delicacy, if prepared properly.",
     "color": "brown",
     "price_postapoc": "25 cent",
     "quench": 0,
     "calories": 133,
-    "vitamins": [ [ "vitC", 11 ], [ "calcium", 0 ], [ "iron", 5 ] ]
+    "vitamins": [
+      ["vitC", 11],
+      ["calcium", 0],
+      ["iron", 5]
+    ]
   },
   {
     "id": "human_sweetbread",
     "type": "COMESTIBLE",
     "copy-from": "sweetbread",
     "looks_like": "sweetbread",
-    "name": { "str": "piece of raw human sweetbread", "str_pl": "pieces of raw human sweetbread" },
+    "name": {
+      "str": "piece of raw human sweetbread",
+      "str_pl": "pieces of raw human sweetbread"
+    },
     "description": "A piece of the thymus or pancreas of a human, which appears to have been extracted too crudely to ever be useful as a transplant.",
-    "material": [ "hflesh" ],
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "material": ["hflesh"],
+    "vitamins": [
+      ["vitC", 11],
+      ["calcium", 0],
+      ["iron", 5],
+      ["meat_allergen", 1],
+      ["human_flesh_vitamin", 1]
+    ]
   },
   {
     "id": "demihuman_sweetbread",
     "type": "COMESTIBLE",
     "copy-from": "sweetbread",
     "looks_like": "sweetbread",
-    "name": { "str": "piece of raw demihuman sweetbread", "str_pl": "pieces of raw demihuman sweetbread" },
+    "name": {
+      "str": "piece of raw demihuman sweetbread",
+      "str_pl": "pieces of raw demihuman sweetbread"
+    },
     "description": "The consistency of this chunk of organ suggests possibly a thymus or pancreas equivalent.",
-    "extend": { "flags": [ "STRICT_HUMANITARIANISM" ] }
+    "extend": { "flags": ["STRICT_HUMANITARIANISM"] }
   },
   {
     "id": "sweetbread_cooked",
     "type": "COMESTIBLE",
     "copy-from": "sweetbread",
     "looks_like": "meat_cooked",
-    "name": { "str": "cooked piece of sweetbread", "str_pl": "cooked pieces of sweetbread" },
+    "name": {
+      "str": "cooked piece of sweetbread",
+      "str_pl": "cooked pieces of sweetbread"
+    },
     "price_postapoc": "50 cent",
     "fun": -1,
     "description": "Normally a delicacy, but this style of preparation is missing something.",
     "parasites": 0,
-    "delete": { "flags": [ "RAW" ] }
+    "delete": { "flags": ["RAW"] }
   },
   {
     "type": "COMESTIBLE",
@@ -1344,12 +1729,17 @@
     "price_postapoc": "10 cent",
     "description": "Blood extracted from a human being.",
     "price": "0 cent",
-    "material": [ "hblood" ],
+    "material": ["hblood"],
     "volume": "250 ml",
     "charges": 1,
     "phase": "liquid",
-    "delete": { "flags": [ "SMOKABLE" ] },
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "delete": { "flags": ["SMOKABLE"] },
+    "vitamins": [
+      ["calcium", 2],
+      ["iron", 12],
+      ["meat_allergen", 1],
+      ["human_flesh_vitamin", 1]
+    ]
   },
   {
     "type": "COMESTIBLE",
@@ -1365,11 +1755,11 @@
     "calories": 1923,
     "description": "A chunk of raw fat.  You could eat it as-is, but it is better used as an ingredient in other foods or projects.",
     "price": "0 cent",
-    "material": [ "flesh" ],
+    "material": ["flesh"],
     "volume": "250 ml",
     "parasites": 32,
     "fun": -20,
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "vitamins": [["meat_allergen", 1]]
   },
   {
     "type": "COMESTIBLE",
@@ -1386,9 +1776,9 @@
     "price": "125 cent",
     "price_postapoc": "2 USD",
     "//": "*May* have been commercially traded.",
-    "material": [ "flesh" ],
+    "material": ["flesh"],
     "volume": "62 ml",
-    "vitamins": [ [ "meat_allergen", 1 ] ],
+    "vitamins": [["meat_allergen", 1]],
     "fun": -18
   },
   {
@@ -1406,7 +1796,7 @@
     "description": "A chunk of raw fat butchered from a heavily-mutated animal.  It smells, if anything, even more disgusting than the rest of the mutant.  There are little puddles of unidentified oils dripping from it.",
     "looks_like": "fat",
     "proportional": { "price": 0.2 },
-    "vitamins": [ [ "mutant_toxin", 360 ] ]
+    "vitamins": [["mutant_toxin", 360]]
   },
   {
     "id": "mutant_tallow",
@@ -1416,7 +1806,7 @@
     "description": "A smooth white block of cleaned and rendered fat, sourced from a mutant animal.  It will remain edible for a very long time, and can be used as an ingredient in many foods and projects.",
     "looks_like": "tallow",
     "proportional": { "price": 0.2 },
-    "vitamins": [ [ "mutant_toxin", 180 ] ]
+    "vitamins": [["mutant_toxin", 180]]
   },
   {
     "id": "mutant_lard",
@@ -1429,10 +1819,16 @@
     "id": "mutant_human_fat",
     "type": "COMESTIBLE",
     "copy-from": "mutant_fat",
-    "name": { "str": "chunk of mutant humanoid fat", "str_pl": "chunks of mutant humanoid fat" },
+    "name": {
+      "str": "chunk of mutant humanoid fat",
+      "str_pl": "chunks of mutant humanoid fat"
+    },
     "description": "Raw fat butchered from a heavily-mutated humanoid.",
-    "material": [ "hflesh" ],
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "material": ["hflesh"],
+    "vitamins": [
+      ["meat_allergen", 1],
+      ["human_flesh_vitamin", 1]
+    ]
   },
   {
     "type": "COMESTIBLE",
@@ -1440,8 +1836,11 @@
     "name": { "str_sp": "mutant humanoid tallow" },
     "copy-from": "mutant_tallow",
     "description": "A smooth white block of cleaned and rendered fat, sourced from a mutant humanoid.  It won't rot for a very long time, and can be used as an ingredient in many foods and projects.",
-    "material": [ "hflesh" ],
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "material": ["hflesh"],
+    "vitamins": [
+      ["meat_allergen", 1],
+      ["human_flesh_vitamin", 1]
+    ]
   },
   {
     "type": "COMESTIBLE",
@@ -1454,23 +1853,26 @@
     "type": "COMESTIBLE",
     "id": "meat_tainted",
     "category": "other",
-    "name": { "str": "chunk of tainted meat", "str_pl": "chunks of tainted meat" },
+    "name": {
+      "str": "chunk of tainted meat",
+      "str_pl": "chunks of tainted meat"
+    },
     "weight": "238 g",
     "color": "red",
     "spoils_in": "4 hours",
-    "use_action": [ "POISON" ],
+    "use_action": ["POISON"],
     "comestible_type": "FOOD",
     "symbol": "%",
     "calories": 174,
     "description": "A piece of meat that's obviously unhealthy.  You could eat it, but it will poison you.",
     "price": "0 cent",
     "price_postapoc": "10 cent",
-    "material": [ "flesh" ],
+    "material": ["flesh"],
     "volume": "250 ml",
     "fun": -10,
-    "flags": [ "TRADER_AVOID", "SMOKABLE", "NUTRIENT_OVERRIDE", "RAW" ],
+    "flags": ["TRADER_AVOID", "SMOKABLE", "NUTRIENT_OVERRIDE", "RAW"],
     "smoking_result": "dry_meat_tainted",
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "vitamins": [["meat_allergen", 1]]
   },
   {
     "type": "COMESTIBLE",
@@ -1481,18 +1883,18 @@
     "snippet_category": "arm_zed_desc",
     "color": "red",
     "symbol": "%",
-    "material": [ "flesh" ],
+    "material": ["flesh"],
     "weight": "3767 g",
     "volume": "4500 ml",
     "comestible_type": "FOOD",
-    "use_action": [ "POISON" ],
+    "use_action": ["POISON"],
     "calories": 278,
     "fun": -20,
-    "vitamins": [ [ "meat_allergen", 1 ] ],
+    "vitamins": [["meat_allergen", 1]],
     "spoils_in": "2 days",
     "price": "0 cent",
     "price_postapoc": "0 cent",
-    "flags": [ "TRADER_AVOID", "NUTRIENT_OVERRIDE", "NO_REPAIR" ]
+    "flags": ["TRADER_AVOID", "NUTRIENT_OVERRIDE", "NO_REPAIR"]
   },
   {
     "type": "COMESTIBLE",
@@ -1503,18 +1905,18 @@
     "snippet_category": "leg_zed_desc",
     "color": "red",
     "symbol": "%",
-    "material": [ "flesh" ],
+    "material": ["flesh"],
     "weight": "11071 g",
     "volume": "12 L",
     "comestible_type": "FOOD",
-    "use_action": [ "POISON" ],
+    "use_action": ["POISON"],
     "calories": 399,
     "fun": -20,
-    "vitamins": [ [ "meat_allergen", 1 ] ],
+    "vitamins": [["meat_allergen", 1]],
     "spoils_in": "2 days",
     "price": "0 cent",
     "price_postapoc": "0 cent",
-    "flags": [ "TRADER_AVOID", "NUTRIENT_OVERRIDE", "NO_REPAIR" ]
+    "flags": ["TRADER_AVOID", "NUTRIENT_OVERRIDE", "NO_REPAIR"]
   },
   {
     "id": "blood_tainted",
@@ -1530,38 +1932,44 @@
     "material": "blood",
     "charges": 1,
     "phase": "liquid",
-    "delete": { "flags": [ "SMOKABLE" ] }
+    "delete": { "flags": ["SMOKABLE"] }
   },
   {
     "type": "COMESTIBLE",
     "id": "tainted_marrow",
     "category": "other",
-    "name": { "str": "portion of tainted bone marrow", "str_pl": "portions of tainted bone marrow" },
+    "name": {
+      "str": "portion of tainted bone marrow",
+      "str_pl": "portions of tainted bone marrow"
+    },
     "weight": "60 g",
     "color": "red",
     "spoils_in": "4 hours",
-    "use_action": [ "POISON" ],
+    "use_action": ["POISON"],
     "comestible_type": "FOOD",
     "symbol": "%",
     "calories": 235,
     "description": "A wet, green chunk of spongy goop extracted from a rotten bone.  You could eat it, but it will poison you.",
     "price": "0 cent",
     "price_postapoc": "10 cent",
-    "material": [ "flesh" ],
+    "material": ["flesh"],
     "volume": "50 ml",
     "fun": -10,
-    "flags": [ "TRADER_AVOID", "RAW" ],
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "flags": ["TRADER_AVOID", "RAW"],
+    "vitamins": [["meat_allergen", 1]]
   },
   {
     "type": "COMESTIBLE",
     "id": "fat_tainted",
     "category": "other",
-    "name": { "str": "chunk of tainted fat", "str_pl": "chunks of tainted fat" },
+    "name": {
+      "str": "chunk of tainted fat",
+      "str_pl": "chunks of tainted fat"
+    },
     "weight": "192 g",
     "color": "yellow",
     "spoils_in": "4 hours",
-    "use_action": [ "POISON" ],
+    "use_action": ["POISON"],
     "comestible_type": "FOOD",
     "symbol": "%",
     "calories": 9,
@@ -1569,11 +1977,11 @@
     "price": "0 cent",
     "price_postapoc": "25 cent",
     "//": "Not for use in edible recipes, and should require ~200% as much as normal for applicable inedible recipes.",
-    "material": [ "flesh" ],
+    "material": ["flesh"],
     "volume": "250 ml",
     "fun": -10,
-    "flags": [ "RAW" ],
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "flags": ["RAW"],
+    "vitamins": [["meat_allergen", 1]]
   },
   {
     "type": "COMESTIBLE",
@@ -1582,7 +1990,7 @@
     "category": "chems",
     "weight": "128 g",
     "color": "light_gray",
-    "use_action": [ "POISON" ],
+    "use_action": ["POISON"],
     "comestible_type": "FOOD",
     "symbol": "%",
     "quench": -10,
@@ -1591,10 +1999,10 @@
     "price": "1 cent",
     "price_postapoc": "12 cent",
     "//": "Not for use in edible/foodsafe recipes.  Inefficiency is handled by tainted tallow recipe requiring 200% as much materials input as regular tallow recipe.",
-    "material": [ "flesh" ],
+    "material": ["flesh"],
     "volume": "62 ml",
     "fun": -18,
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "vitamins": [["meat_allergen", 1]]
   },
   {
     "type": "COMESTIBLE",
@@ -1602,7 +2010,12 @@
     "copy-from": "small_stomach_boiled",
     "name": { "str": "large boiled stomach" },
     "description": "A boiled stomach from an animal, nothing else.  It looks all but appetizing.",
-    "proportional": { "price": 2.0, "calories": 2.0, "volume": 2.0, "weight": 2.0 },
+    "proportional": {
+      "price": 2.0,
+      "calories": 2.0,
+      "volume": 2.0,
+      "weight": 2.0
+    },
     "fun": -5
   },
   {
@@ -1611,8 +2024,11 @@
     "copy-from": "stomach_boiled",
     "name": { "str": "boiled large human stomach" },
     "description": "A boiled stomach from a large humanoid creature, nothing else.  It looks all but appetizing.",
-    "material": [ "hflesh" ],
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "material": ["hflesh"],
+    "vitamins": [
+      ["meat_allergen", 1],
+      ["human_flesh_vitamin", 1]
+    ]
   },
   {
     "type": "COMESTIBLE",
@@ -1627,11 +2043,11 @@
     "description": "A small boiled stomach from an animal, nothing else.  It looks all but appetizing.",
     "price": "3 USD 75 cent",
     "price_postapoc": "10 cent",
-    "material": [ "flesh" ],
+    "material": ["flesh"],
     "volume": "250 ml",
-    "flags": [ "EATEN_HOT" ],
+    "flags": ["EATEN_HOT"],
     "fun": -3,
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "vitamins": [["meat_allergen", 1]]
   },
   {
     "type": "COMESTIBLE",
@@ -1639,8 +2055,11 @@
     "copy-from": "small_stomach_boiled",
     "name": { "str": "boiled human stomach" },
     "description": "A small boiled stomach from a human, nothing else.  It looks all but appetizing.",
-    "material": [ "hflesh" ],
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "material": ["hflesh"],
+    "vitamins": [
+      ["meat_allergen", 1],
+      ["human_flesh_vitamin", 1]
+    ]
   },
   {
     "type": "COMESTIBLE",
@@ -1658,10 +2077,10 @@
     "description": "A carefully folded raw skin harvested from an animal.  You can cure it for storage and tanning, or eat it if you're desperate enough.",
     "price": "3 USD 30 cent",
     "price_postapoc": "20 cent",
-    "material": [ "flesh" ],
-    "flags": [ "TRADER_AVOID" ],
+    "material": ["flesh"],
+    "flags": ["TRADER_AVOID"],
     "fun": -12,
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "vitamins": [["meat_allergen", 1]]
   },
   {
     "type": "COMESTIBLE",
@@ -1669,7 +2088,7 @@
     "copy-from": "raw_leather",
     "name": { "str_sp": "tainted hide" },
     "spoils_in": "6 hours",
-    "use_action": [ "POISON" ],
+    "use_action": ["POISON"],
     "price_postapoc": "10 cent",
     "description": "A carefully folded poisonous raw skin harvested from an unnatural creature.  You can cure it for storage and tanning.",
     "price": "0 cent"
@@ -1683,7 +2102,7 @@
     "volume": "5 ml",
     "spoils_in": "3 hours",
     "calories": 3,
-    "use_action": [ "POISON" ],
+    "use_action": ["POISON"],
     "price_postapoc": "0 cent",
     "description": "Shredded scraps of raw skin from an unnatural creature.  It can't have much use.",
     "price": "0 cent"
@@ -1694,8 +2113,11 @@
     "copy-from": "raw_leather",
     "name": { "str_sp": "raw human skin" },
     "description": "A carefully folded raw skin harvested from a human.  You can cure it for storage and tanning, or eat it if you're desperate enough.",
-    "material": [ "hflesh" ],
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "material": ["hflesh"],
+    "vitamins": [
+      ["meat_allergen", 1],
+      ["human_flesh_vitamin", 1]
+    ]
   },
   {
     "type": "COMESTIBLE",
@@ -1703,9 +2125,9 @@
     "copy-from": "raw_leather",
     "name": { "str_sp": "raw demihuman skin" },
     "description": "A carefully folded raw skin harvested from a demihuman.  You can cure it for storage and tanning, or eat it if you're desperate enough.",
-    "material": [ "flesh" ],
-    "extend": { "flags": [ "STRICT_HUMANITARIANISM" ] },
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "material": ["flesh"],
+    "extend": { "flags": ["STRICT_HUMANITARIANISM"] },
+    "vitamins": [["meat_allergen", 1]]
   },
   {
     "type": "COMESTIBLE",
@@ -1723,10 +2145,10 @@
     "description": "A carefully folded raw skin harvested from a fur-bearing animal.  It still has the fur attached.  You can cure it for storage and tanning, or eat it if you're desperate enough.",
     "price": "3 USD 30 cent",
     "price_postapoc": "1 USD",
-    "material": [ "fur", "flesh" ],
-    "flags": [ "NO_SALVAGE", "TRADER_AVOID" ],
+    "material": ["fur", "flesh"],
+    "flags": ["NO_SALVAGE", "TRADER_AVOID"],
     "fun": -24,
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "vitamins": [["meat_allergen", 1]]
   },
   {
     "type": "COMESTIBLE",
@@ -1734,7 +2156,7 @@
     "name": { "str_sp": "tainted pelt" },
     "copy-from": "raw_fur",
     "spoils_in": "6 hours",
-    "use_action": [ "POISON" ],
+    "use_action": ["POISON"],
     "description": "A carefully folded raw skin harvested from a fur-bearing unnatural creature.  It still has the fur attached and is poisonous.  You can cure it for storage and tanning.",
     "price": "0 cent"
   },
@@ -1744,8 +2166,11 @@
     "copy-from": "raw_fur",
     "name": { "str_sp": "raw human pelt" },
     "description": "A carefully folded raw skin harvested from a fur-bearing mutant human.  It still has the fur attached.  You can cure it for storage and tanning, or eat it if you're desperate enough.",
-    "material": [ "fur", "hflesh" ],
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "material": ["fur", "hflesh"],
+    "vitamins": [
+      ["meat_allergen", 1],
+      ["human_flesh_vitamin", 1]
+    ]
   },
   {
     "type": "COMESTIBLE",
@@ -1761,11 +2186,14 @@
     "description": "A thick mass of flesh superficially resembling a mammalian heart.  It's the size of your fist and covered in dimpled grooves.",
     "price": "30 USD",
     "price_postapoc": "1 USD",
-    "material": [ "flesh" ],
+    "material": ["flesh"],
     "volume": "500 ml",
     "fun": -25,
-    "vitamins": [ [ "mutagen_beast", 250 ], [ "meat_allergen", 1 ] ],
-    "flags": [ "PREDATOR_FUN" ]
+    "vitamins": [
+      ["mutagen_beast", 250],
+      ["meat_allergen", 1]
+    ],
+    "flags": ["PREDATOR_FUN"]
   },
   {
     "type": "COMESTIBLE",
@@ -1780,11 +2208,14 @@
     "description": "A thick, hulking mass of flesh that's easily the size of your head, superficially resembling a mammalian heart and covered in ribbed grooves.  It's still full of whatever passes for blood in jabberwocks, and is heavy in your hands.  After everything you've seen lately, you can't help but remember old sayings about eating the hearts of your enemies…",
     "price": "65 USD",
     "price_postapoc": "5 USD",
-    "material": [ "flesh" ],
+    "material": ["flesh"],
     "volume": "1 L",
     "fun": -25,
-    "vitamins": [ [ "mutagen_beast", 300 ], [ "meat_allergen", 1 ] ],
-    "flags": [ "PREDATOR_FUN" ]
+    "vitamins": [
+      ["mutagen_beast", 300],
+      ["meat_allergen", 1]
+    ],
+    "flags": ["PREDATOR_FUN"]
   },
   {
     "type": "COMESTIBLE",
@@ -1799,22 +2230,27 @@
     "description": "A huge strip of muscle - all that remains of a putrid heart that has been sliced open and drained of blood.  It could be eaten if you're hungry, but looks *disgusting*.",
     "price": "10 USD",
     "price_postapoc": "25 cent",
-    "material": [ "flesh" ],
-    "use_action": [ "POISON" ],
+    "material": ["flesh"],
+    "use_action": ["POISON"],
     "volume": "250 ml",
     "fun": -30,
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "vitamins": [["meat_allergen", 1]]
   },
   {
     "id": "meat_frond",
     "copy-from": "meat_tainted",
     "type": "COMESTIBLE",
     "name": { "str_sp": "alien fronds" },
-    "use_action": [ "POISON" ],
+    "use_action": ["POISON"],
     "description": "The fleshy fronds harvested from an alien plant.  Eating these membranous leaves and gut-like stems is likely a terrible idea, and yet they have a paradoxically pleasant and inviting sweet smell.  Might be non-vegan.",
     "fun": 15,
     "stim": 3,
-    "vitamins": [ [ "vitC", 2 ], [ "calcium", 0 ], [ "iron", 8 ], [ "mutant_toxin", 8 ] ]
+    "vitamins": [
+      ["vitC", 2],
+      ["calcium", 0],
+      ["iron", 8],
+      ["mutant_toxin", 8]
+    ]
   },
   {
     "id": "leech_flower",
@@ -1850,7 +2286,12 @@
     "name": { "str": "leech bark", "str_pl": "scraps of leech bark" },
     "description": "Dry and tough bark matter harvested from an alien plant.  It is slightly translucent, and if placed against the light you can distinguish glistening blue veins running through it.",
     "price_postapoc": "10 cent",
-    "vitamins": [ [ "vitC", 0 ], [ "calcium", 2 ], [ "iron", 8 ], [ "mutant_toxin", 12 ] ]
+    "vitamins": [
+      ["vitC", 0],
+      ["calcium", 2],
+      ["iron", 8],
+      ["mutant_toxin", 12]
+    ]
   },
   {
     "id": "demihuman_stomach",
@@ -1858,9 +2299,9 @@
     "type": "COMESTIBLE",
     "name": { "str": "demihuman stomach" },
     "description": "The stomach of an intelligent demihuman.  It is surprisingly durable.",
-    "material": [ "flesh" ],
-    "extend": { "flags": [ "STRICT_HUMANITARIANISM" ] },
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "material": ["flesh"],
+    "extend": { "flags": ["STRICT_HUMANITARIANISM"] },
+    "vitamins": [["meat_allergen", 1]]
   },
   {
     "id": "demihuman_stomach_large",
@@ -1868,18 +2309,26 @@
     "type": "COMESTIBLE",
     "name": { "str": "large demihuman stomach" },
     "description": "The stomach of a large humanoid creature.  It is surprisingly durable.",
-    "proportional": { "weight": 2.0, "volume": 2.0, "price": 1.5, "calories": 2.0 },
-    "extend": { "flags": [ "STRICT_HUMANITARIANISM" ] }
+    "proportional": {
+      "weight": 2.0,
+      "volume": 2.0,
+      "price": 1.5,
+      "calories": 2.0
+    },
+    "extend": { "flags": ["STRICT_HUMANITARIANISM"] }
   },
   {
     "id": "demihuman_fat",
     "copy-from": "fat",
     "type": "COMESTIBLE",
-    "name": { "str": "chunk of demihuman fat", "str_pl": "chunks of demihuman fat" },
+    "name": {
+      "str": "chunk of demihuman fat",
+      "str_pl": "chunks of demihuman fat"
+    },
     "description": "Raw fat harvested from a demihuman body.",
-    "material": [ "flesh" ],
-    "extend": { "flags": [ "STRICT_HUMANITARIANISM" ] },
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "material": ["flesh"],
+    "extend": { "flags": ["STRICT_HUMANITARIANISM"] },
+    "vitamins": [["meat_allergen", 1]]
   },
   {
     "type": "COMESTIBLE",
@@ -1889,9 +2338,9 @@
     "description": "A smooth white block of cleaned and rendered demihuman fat.  It will remain edible for a very long time, and can be used as an ingredient in many foods and projects.",
     "price": "5 USD",
     "//": "*May* have been commercially traded.",
-    "material": [ "flesh" ],
-    "extend": { "flags": [ "STRICT_HUMANITARIANISM" ] },
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "material": ["flesh"],
+    "extend": { "flags": ["STRICT_HUMANITARIANISM"] },
+    "vitamins": [["meat_allergen", 1]]
   },
   {
     "type": "COMESTIBLE",
@@ -1899,7 +2348,7 @@
     "name": { "str_sp": "demihuman lard" },
     "copy-from": "lard",
     "description": "A smooth white block of dry-rendered demihuman fat.  It will remain edible for a very long time, and can be used as an ingredient in many foods and projects.",
-    "extend": { "flags": [ "STRICT_HUMANITARIANISM" ] }
+    "extend": { "flags": ["STRICT_HUMANITARIANISM"] }
   },
   {
     "id": "demihuman_flesh",
@@ -1908,9 +2357,9 @@
     "cooks_like": "demihuman_cooked",
     "name": { "str_sp": "demihuman flesh" },
     "description": "Raw meat butchered from a demihuman body.",
-    "material": [ "flesh" ],
-    "extend": { "flags": [ "STRICT_HUMANITARIANISM" ] },
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "material": ["flesh"],
+    "extend": { "flags": ["STRICT_HUMANITARIANISM"] },
+    "vitamins": [["meat_allergen", 1]]
   },
   {
     "id": "demihuman_blood",
@@ -1925,9 +2374,10 @@
     "looks_like": "blood",
     "symbol": "~",
     "material": "blood",
+    "extend": { "flags": ["STRICT_HUMANITARIANISM"] },
     "charges": 1,
     "phase": "liquid",
-    "delete": { "flags": [ "SMOKABLE" ] }
+    "delete": { "flags": ["SMOKABLE"] }
   },
   {
     "id": "demihuman_cooked",
@@ -1938,8 +2388,8 @@
     "proportional": { "price": 1.5 },
     "parasites": 0,
     "fun": 0,
-    "extend": { "flags": [ "STRICT_HUMANITARIANISM" ] },
-    "delete": { "flags": [ "RAW" ] }
+    "extend": { "flags": ["STRICT_HUMANITARIANISM"] },
+    "delete": { "flags": ["RAW"] }
   },
   {
     "type": "COMESTIBLE",
@@ -1948,9 +2398,9 @@
     "name": { "str": "boiled demihuman stomach" },
     "calories": 240,
     "description": "A boiled stomach from a demihuman, nothing else.  It looks all but appetizing.",
-    "material": [ "flesh" ],
-    "extend": { "flags": [ "STRICT_HUMANITARIANISM" ] },
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "material": ["flesh"],
+    "extend": { "flags": ["STRICT_HUMANITARIANISM"] },
+    "vitamins": [["meat_allergen", 1]]
   },
   {
     "type": "COMESTIBLE",
@@ -1958,9 +2408,9 @@
     "copy-from": "small_stomach_boiled",
     "name": { "str": "boiled demihuman stomach" },
     "description": "A small boiled stomach from a demihuman, nothing else.  It looks all but appetizing.",
-    "material": [ "flesh" ],
-    "extend": { "flags": [ "STRICT_HUMANITARIANISM" ] },
-    "vitamins": [ [ "meat_allergen", 1 ] ]
+    "material": ["flesh"],
+    "extend": { "flags": ["STRICT_HUMANITARIANISM"] },
+    "vitamins": [["meat_allergen", 1]]
   },
   {
     "id": "meat_fried",
@@ -1968,9 +2418,21 @@
     "type": "COMESTIBLE",
     "name": { "str_sp": "fried meat" },
     "conditional_names": [
-      { "type": "VITAMIN", "condition": "human_flesh_vitamin", "name": { "str_sp": "fried fool" } },
-      { "type": "FLAG", "condition": "STRICT_HUMANITARIANISM", "name": { "str_sp": "fried familiar" } },
-      { "type": "COMPONENT_ID_SUBSTRING", "condition": "mutant", "name": { "str_sp": "%s, mutant" } }
+      {
+        "type": "VITAMIN",
+        "condition": "human_flesh_vitamin",
+        "name": { "str_sp": "fried fool" }
+      },
+      {
+        "type": "FLAG",
+        "condition": "STRICT_HUMANITARIANISM",
+        "name": { "str_sp": "fried familiar" }
+      },
+      {
+        "type": "COMPONENT_ID_SUBSTRING",
+        "condition": "mutant",
+        "name": { "str_sp": "%s, mutant" }
+      }
     ],
     "description": "Delicious oil-fried meat.",
     "color": "brown",
@@ -1978,7 +2440,7 @@
     "quench": -1,
     "fun": 6,
     "weight": "298 g",
-    "flags": [ "EATEN_HOT" ]
+    "flags": ["EATEN_HOT"]
   },
   {
     "type": "COMESTIBLE",
@@ -1989,17 +2451,17 @@
     "color": "pink",
     "spoils_in": "90 days",
     "//": "It's so alien few micro organisms are able to degrade it.",
-    "use_action": [ "POISON" ],
+    "use_action": ["POISON"],
     "comestible_type": "FOOD",
     "symbol": "%",
     "calories": 0,
     "description": "A piece of mi-go meat that's obviously alien, given its pink color, odd odor, and strange texture.  It would not be advisable to try to eat it.",
     "price": "0 cent",
     "price_postapoc": "0 cent",
-    "material": [ "mi-go_flesh" ],
+    "material": ["mi-go_flesh"],
     "volume": "250 ml",
     "fun": -10,
-    "flags": [ "TRADER_AVOID", "NUTRIENT_OVERRIDE" ]
+    "flags": ["TRADER_AVOID", "NUTRIENT_OVERRIDE"]
   },
   {
     "id": "blood_mi-go",
@@ -2021,14 +2483,14 @@
     "name": { "str": "mi-go bone" },
     "weight": "350 g",
     "color": "light_green",
-    "use_action": [ "POISON" ],
+    "use_action": ["POISON"],
     "comestible_type": "FOOD",
     "symbol": "%",
     "calories": 0,
     "description": "An unrecognized structural element from inside a mi-go.  It's strong and springy, somewhat like cartilage, but a bit stiffer.  It's also a deep green color, and might possibly serve a scaffolding role, like bone.",
     "price": "0 cent",
     "price_postapoc": "0 cent",
-    "material": [ "mi-go_bone" ],
+    "material": ["mi-go_bone"],
     "volume": "250 ml",
     "fun": -10
   },
@@ -2036,7 +2498,10 @@
     "id": "poultry",
     "copy-from": "flesh",
     "type": "COMESTIBLE",
-    "name": { "str": "chunk of poultry meat", "str_pl": "chunks of poultry meat" },
+    "name": {
+      "str": "chunk of poultry meat",
+      "str_pl": "chunks of poultry meat"
+    },
     "description": "A chunk of raw meat from a bird.  Dangerous to eat as-is.",
     "weight": "104 g",
     "volume": "100 ml",
@@ -2047,20 +2512,34 @@
     "//": "Scaled from 100g skin on, boneless, wild duck meat from USDA database",
     "fun": -18,
     "cooks_like": "meat_cooked",
-    "vitamins": [ [ "calcium", "5 mg" ], [ "iron", "5 mg" ] ],
-    "flags": [ "SMOKABLE", "RAW", "PREDATOR_FUN" ],
+    "vitamins": [
+      ["calcium", "5 mg"],
+      ["iron", "5 mg"]
+    ],
+    "flags": ["SMOKABLE", "RAW", "PREDATOR_FUN"],
     "smoking_result": "poultry_smoked"
   },
   {
     "id": "poultry_scrap",
     "copy-from": "poultry",
     "type": "COMESTIBLE",
-    "name": { "str": "scrap of poultry meat", "str_pl": "scraps of poultry meat" },
+    "name": {
+      "str": "scrap of poultry meat",
+      "str_pl": "scraps of poultry meat"
+    },
     "description": "A tiny scrap of raw meat from a bird.  You figure that one of these should be about one fifth of a portion.  Dangerous to eat as-is.",
     "cooks_like": "meat_scrap_cooked",
-    "proportional": { "weight": 0.2, "volume": 0.2, "price_postapoc": 0.2, "calories": 0.2 },
-    "delete": { "flags": [ "SMOKABLE" ] },
-    "vitamins": [ [ "calcium", "1 mg" ], [ "iron", "1 mg" ] ]
+    "proportional": {
+      "weight": 0.2,
+      "volume": 0.2,
+      "price_postapoc": 0.2,
+      "calories": 0.2
+    },
+    "delete": { "flags": ["SMOKABLE"] },
+    "vitamins": [
+      ["calcium", "1 mg"],
+      ["iron", "1 mg"]
+    ]
   },
   {
     "id": "poultry_cooked",
@@ -2076,8 +2555,11 @@
     "parasites": 0,
     "calories": 220,
     "fun": 1,
-    "vitamins": [ [ "calcium", "5 mg" ], [ "iron", "5 mg" ] ],
-    "flags": [ "EATEN_HOT", "NUTRIENT_OVERRIDE" ]
+    "vitamins": [
+      ["calcium", "5 mg"],
+      ["iron", "5 mg"]
+    ],
+    "flags": ["EATEN_HOT", "NUTRIENT_OVERRIDE"]
   },
   {
     "id": "poultry_pepper",
@@ -2093,32 +2575,47 @@
     "fun": 3,
     "parasites": 0,
     "calories": 220,
-    "vitamins": [ [ "calcium", "5 mg" ], [ "iron", "5 mg" ] ],
-    "flags": [ "EATEN_HOT", "NUTRIENT_OVERRIDE" ]
+    "vitamins": [
+      ["calcium", "5 mg"],
+      ["iron", "5 mg"]
+    ],
+    "flags": ["EATEN_HOT", "NUTRIENT_OVERRIDE"]
   },
   {
     "id": "poultry_scrap_cooked",
     "copy-from": "poultry_scrap",
     "type": "COMESTIBLE",
-    "name": { "str": "cooked scrap of poultry", "str_pl": "cooked scraps of poultry" },
+    "name": {
+      "str": "cooked scrap of poultry",
+      "str_pl": "cooked scraps of poultry"
+    },
     "description": "A tiny scrap of cooked meat from a bird.  Not very filling, and a bit bland.",
     "parasites": 0,
     "fun": 1,
-    "flags": [ "NUTRIENT_OVERRIDE" ],
+    "flags": ["NUTRIENT_OVERRIDE"],
     "calories": 55,
-    "vitamins": [ [ "calcium", "1 mg" ], [ "iron", "1 mg" ] ]
+    "vitamins": [
+      ["calcium", "1 mg"],
+      ["iron", "1 mg"]
+    ]
   },
   {
     "id": "poultry_scrap_pepper",
     "copy-from": "poultry_scrap",
     "type": "COMESTIBLE",
-    "name": { "str": "cooked pepper scrap of poultry", "str_pl": "cooked pepper scraps of poultry" },
+    "name": {
+      "str": "cooked pepper scrap of poultry",
+      "str_pl": "cooked pepper scraps of poultry"
+    },
     "description": "A tiny scrap of cooked bird meat, seasoned with simple spices.  Makes for a nice high-protein snack.",
     "parasites": 0,
     "fun": 3,
-    "flags": [ "NUTRIENT_OVERRIDE" ],
+    "flags": ["NUTRIENT_OVERRIDE"],
     "calories": 55,
-    "vitamins": [ [ "calcium", "1 mg" ], [ "iron", "1 mg" ] ]
+    "vitamins": [
+      ["calcium", "1 mg"],
+      ["iron", "1 mg"]
+    ]
   },
   {
     "id": "poultry_smoked",
@@ -2130,17 +2627,20 @@
     "color": "brown",
     "calories": 220,
     "price_postapoc": "4 USD",
-    "flags": [ "EATEN_HOT", "SMOKED", "SMOKABLE" ],
+    "flags": ["EATEN_HOT", "SMOKED", "SMOKABLE"],
     "smoking_result": "dry_poultry"
   },
   {
     "id": "mutflesh",
     "copy-from": "meat",
     "type": "COMESTIBLE",
-    "name": { "str": "chunk of mutagenic flesh", "str_pl": "chunks of mutagenic flesh" },
+    "name": {
+      "str": "chunk of mutagenic flesh",
+      "str_pl": "chunks of mutagenic flesh"
+    },
     "description": "A chunk of raw meat with a weirdly off-green coloration.  You could eat it as-is, but are you sure you want to?",
-    "flags": [ "RAW", "TRADER_AVOID" ],
+    "flags": ["RAW", "TRADER_AVOID"],
     "smoking_result": " ",
-    "vitamins": [ [ "mutagenic_slurry", 12 ] ]
+    "vitamins": [["mutagenic_slurry", 12]]
   }
 ]
diff --git a/data/json/items/resources/bone.json b/data/json/items/resources/bone.json
index d002f41b29..9e3c1051b0 100644
--- a/data/json/items/resources/bone.json
+++ b/data/json/items/resources/bone.json
@@ -60,7 +60,7 @@
     "name": { "str": "human bone" },
     "description": "A bone from a human being.  Could be used to make some stuff, if you're feeling sufficiently ghoulish.",
     "//": "No extend support for vitamins, we need to overwrite",
-    "vitamins": [ [ "calcium", "25588 mg" ], [ "meat_allergen", 1 ], [ "human_flesh_vitamin", 100 ] ]
+    "vitamins": [ [ "calcium", "25588 mg" ], [ "meat_allergen", 1 ], [ "human_flesh_vitamin", 1 ] ]
   },
   {
     "type": "GENERIC",
-- 
2.45.2

